docs/how_tos/manual_testing.rst

```diff
@@ -0,0 +1,39 @@
+Manual testing
+==============
+
+The producer can be tested manually against a Kafka running in devstack.
+
+#. Create a "unit test" in one of the test files that will actually call Kafka. For example, this could be added to the end of ``edx_event_bus_kafka/publishing/test_event_producer.py``::
+
+    import random
+
+    def test_actually_send_to_event_bus():
+        signal = openedx_events.learning.signals.SESSION_LOGIN_COMPLETED
+        # Make events distinguishable
+        id = random.randrange(1000)
+        event_data = {
+            'user': UserData(
+                id=id,
+                is_active=True,
+                pii=UserPersonalData(
+                    username=f'foobob_{id:03}',
+                    email='bob@foo.example',
+                    name="Bob Foo",
+                )
+            )
+        }
+
+        print(f"Sending event with random user ID {id}.")
+        with override_settings(
+                SCHEMA_REGISTRY_URL='http://edx.devstack.schema-registry:8081',
+                KAFKA_BOOTSTRAP_SERVERS='edx.devstack.kafka:29092',
+        ):
+            ep.send_to_event_bus(signal, 'user_stuff', 'user.id', event_data)
+
+#. Make or refresh a copy of this repo where it can be seen from inside devstack: ``rsync -sax -delete ./ ../src/event-bus-kafka/``
```

Comment:
[not blocking] Should we update this to point to...whatever documentation we have around how to use edx-platform plugins in general?

Reply:
I couldn't find anything, but I didn't look _super_ hard. But I think it would be worth investigating instead if we can call a test cluster more directly, not requiring the test to be run inside of devstack.